### PR TITLE
add multiple schema examples; tag issues

### LINKED ISSUE #121: Support multiple schemas applying to different portions of a VC
Consider a credential pointing to a single VC JSON Schema identified `https://example.org/examples/verifiable-credential-schema.json`

```json
{
  "@context": [
    "https://www.w3.org/2018/credentials/v1",
    "https://www.w3.org/2018/credentials/examples/v1"
  ],
  "id": "http://example.edu/credentials/3732",
  "type": ["VerifiableCredential", "UniversityDegreeCredential"],
  "issuer": "https://example.edu/issuers/14",
  "issuanceDate": "2010-01-01T19:23:24Z",
  "credentialSubject": {
    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
    "degree": {
      "type": "BachelorDegree",
      "name": "Bachelor of Science and Arts"
    }
  },
  "credentialSchema": {
    "id": "https://example.org/examples/verifiable-credential-schema.json",
    "type": "CredentialSchema2022"
  }
}
```

This schema resolves to a document such as

```json
{
    "type": "https://w3c-ccg.github.io/vc-json-schemas/schema/2.0/schema.json",
    "version": "1.0",
    "id": "did:example:MDP8AsFhHzhwUvGNuYkX7T/06e126d1-fa44-4882-a243-1e326fbe21db?version=1.0",
    "name": "Email",
    "author": "did:example:MDP8AsFhHzhwUvGNuYkX7T",
    "authored": "2022-05-05T00:00:00+00:00",
    "path": "$.credentialSubject",
    "schemas":
    [
        {
            "path": "$.credentialSubject",
            "schema":
            {
                "$id": "name-schema-1.0",
                "$schema": "https://json-schema.org/draft/2019-09/schema",
                "description": "Name",
                "type": "object",
                "properties":
                {
                    "firstName":
                    {
                        "type": "string"
                    },
                    "lastName":
                    {
                        "type": "string"
                    }
                },
                "required":
                [
                    "firstName",
                    "lastName"
                ],
                "additionalProperties": false
            }
        },
        {
            "path": "$.credentialSubject",
            "schema":
            {
                "$id": "email-schema-1.0",
                "$schema": "https://json-schema.org/draft/2019-09/schema",
                "description": "Email",
                "type": "object",
                "properties":
                {
                    "emailAddress":
                    {
                        "type": "string",
                        "format": "email"
                    }
                },
                "required":
                [
                    "emailAddress"
                ],
                "additionalProperties": false
            }
        }
    ]
}
```

In this way a single VC JSON Schema can embed multiple schemas in a single document which apply to the same (or perhaps different) part(s) of a credential.

### PR DESCRIPTION
**Pull request by decentralgabe**
fixes #121

### CODE REVIEW

File: v2/index.bs [hunk 1]
```diff
@@ -418,37 +372,182 @@ A [=Credential Schema=] <i>MUST</i> have a `schema` property. The `schema` prope
...
+
+# Processing # {#processing}
+
+There is wide support for JSON Schema processing and support for [[JSON-SCHEMA-IMPLMENTATIONS]] can be found in most programming languages.
+
+To process a [=Verifiable Credential=] against a [=Credential Schema=] one must first extract the schema from the `schema` property of a [=Credential Schema=] and next apply [validation](#processing_validation).
+
+## Validation ## {#processing_validation}
+
+What determines the validity of a [=Verifiable Credential=] is at the discretion of <i>verifier</i> for a given credential. Validity could mean a a valid [=proof=], a valid [=proof=] and validation against a given [=schema=], a valid [=proof=], valid <i>credentialStatus</i>, trusted <i>issuer</i>, or any other set of conditions. The [[VC-DATA-MODEL]]'s section on <a href="https://w3c.github.io/vc-data-model/#validity-checks">Validity Checks</a> offers guidance on determining whether a [=Verifiable Credential=] should be considered valid.
```
Comment (andresuribe87) [inline v2/index.bs:505]: The statement `a valid [=proof=]` is repeated. I suggest removing.

**Review by andresuribe87 — APPROVED** [review]
Nice!

**Comment by TallTed** [comment]
@decentralgabe @andresuribe87 —

I submit that [`a valid [=proof=]` was intentionally repeated](https://github.com/w3c-ccg/vc-json-schemas/pull/123#discussion_r1117802846) in [line 799 of index.html](https://github.com/w3c-ccg/vc-json-schemas/blob/584309f3eb046984587ee72510969499e0904d49/docs/index.html#L799), as it was part of multiple condition sets.

Instead, the punctuation of that line should have been changed, to this --
```suggestion
What determines the validity of a [=Verifiable Credential=] is at the discretion of <i>verifier</i> for a given credential. Validity could mean any of: a valid [=proof=]; a valid [=proof=] and validation against a given [=schema=]; a valid [=proof=], valid <i>credentialStatus</i>, and trusted <i>issuer</i>; or any other set of conditions. The [[VC-DATA-MODEL]]'s section on <a href="https://w3c.github.io/vc-data-model/#validity-checks">Validity Checks</a> offers guidance on determining whether a [=Verifiable Credential=] should be considered valid.
```

This PR, which was [created at 18:21 ET on Friday 2023-02-24 at 18:21 ET](https://github.com/w3c-ccg/vc-json-schemas/pull/123#issue-1599396786), should not have been [merged at 18:54 ET](https://github.com/w3c-ccg/vc-json-schemas/pull/123#event-8606615620) that same day!

There are additional issues in the content of this PR, which I don't feel like turning into my own PR at this time, but would have been easily added via requested change on your PR.

**Comment by decentralgabe** [comment]
@TallTed please leave your comments here and I will create another PR. I have followed the pattern of merging quickly and being open to additional changes since we're in an early state with the draft. If you prefer I will leave the PRs open for a few days going forward. 

This is to say we do value your (and everyone's) feedback and will make sure to incorporate it.

**Comment by TallTed** [comment]
> If you prefer I will leave the PRs open for a few days going forward.

Yes, please. Not just for me! Generally speaking, especially where we are in the process (with plenty of time to CR), I suggest that Pulls be left open for 3–4 days or even a week -- such that people who have whatever work days (and weekends/holidays) have an opportunity to review and provide input before the Pull is merged.

I don't think GitHub will let me run a Review on this now, but I will review your changes and put in some number of suggestions.

**Comment by TallTed** [comment]
(Should suggestions be against the `.html` or the `.bs`?)

**Comment by decentralgabe** [comment]
@TallTed the `.bs` is the source of truth from which html is generated, so there is preferred.

**Pull request merged**